Gemfile

```diff
@@ -27,10 +27,10 @@ gem 'git', '~> 1.2.5'
 gem 'oily_png', '~> 1.1.1'
 gem 'nokogiri', '~> 1.5.10'
 gem 'therubyracer', platforms: :ruby, require: linux_only('therubyracer')
-gem 'aweplug', github: 'awestruct/aweplug', ref: 'f4dd939f5a17d1946cebe5339411c411a8f483e3'
-# gem 'aweplug', github: 'Dantheman720/aweplug', :branch => 'DEVELOPER-1241'
+# gem 'aweplug', github: 'awestruct/aweplug', ref: 'f4dd939f5a17d1946cebe5339411c411a8f483e3'
+gem 'aweplug', github: 'Dantheman720/aweplug', ref: '4f0472883ebb8c6064546760c946b367336475fd'
```

Comment:
Aha, I see it's now in https://github.com/awestruct/aweplug/commits/jboss_org. Can you update this file to point to that please.